packages/node_modules/@webex/plugin-meetings/src/peer-connection-manager/index.js

```diff
@@ -443,17 +371,16 @@ PC.close = (pc) => {
   // pc.close() fails on firefox on network changes and gives a Dom exception
   // To avoid this we have added a try catch block.
   // Please refer to https://bugzilla.mozilla.org/show_bug.cgi?id=1274407 for more information
-  console.info('PC: close() -> closing the mediaPeerConnection');
+
+  if(pc && pc.connectionState === 'closed') {
```

Comment:
Is 'closed' a constant? Or is this the only spot where this scope is used?

Reply:
no it should be a constant